### DISCUSSION TITLE
Very long asset filenames collapses assets table early when viewport scaled

### DISCUSSION THREAD

**Discussion by howells** [craftcms/cms#8541] [Ideas]
My client seems to love using extremely long filenames, which means (I assume) the table view collapses quite early - https://cl.ly/rAYL - I sort of feel it would be nicer if the filenames were just truncated to avoid this, since the collapsed view is much harder to work with on reasonably large viewports.

### Additional info

- Craft version: 3.0.4

**Comment by brandonkelly** [comment]
Related: #2731

**Comment by peirix** [comment]
+1
